compiler/mio-circle08/src/Reader.cpp

```diff
@@ -45,6 +46,28 @@ Reader::Reader(const ::circle::Model *model)
   }
 }
 
+Reader::Reader(const ::circle::Model *model, const std::vector<char> *rawdata)
```

Comment:
(optional)
```suggestion
Reader::Reader(const ::circle::Model *model, const std::vector<char> *rawdata): Reader(model)
{
  _rawdata = rawdata;
}
```

Reply:
Ah, I'm planning to remove old `Reader::Reader`